core/src/main/java/org/bitcoinj/wallet/DerivationPathFactory.java

```diff
@@ -23,6 +27,22 @@ public DerivationPathFactory(NetworkParameters params) {
                 .build();
     }
 
+    /** blockchain identity topup funding derivation path
+     * m/9'/5'/12' (mainnet)
+     * m/9'/1'/12' (testnet, devnets)
+     */
+    public ImmutableList<ChildNumber> blockchainIdentityTopupFundingDerivationPath() {
+        return ImmutableList.<ChildNumber>builder()
+                .add(FEATURE_PURPOSE)
+                .add(coinType)
+                .add(new ChildNumber(13, true))
```

Comment:
Why it's 13 here and 12 on the comment?

Reply:
Good catch!

Reply:
https://github.com/dashevo/dashj/pull/81/commits/03114b9f818b226a8ce0193fcb6fe0bdde5322d8 - fixed here